Assets/Script/Controllers/OverviewController.cs

```diff
@@ -0,0 +1,117 @@
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class OverviewController : MonoBehaviour
+{
+    [Header("Body Parts")]
+    [SerializeField] private ScriptableObject[] bodyConfigs;
+    [SerializeField] private ScriptableObject[] cannonConfigs;
+    [SerializeField] private ScriptableObject[] turretConfigs;
+    [SerializeField] private ScriptableObject[] wheelConfigs;
+    [SerializeField] private TankConfig playerTank;
+
+    [Header("Mesh Filters")]
+    [SerializeField] private MeshFilter bodyMesh;
+    [SerializeField] private MeshFilter cannonMesh;
+    [SerializeField] private MeshFilter turretMesh;
+    [SerializeField] private MeshFilter wheelMesh;
+
+    [Header("UI Texts")]
+    [SerializeField] private TMP_Text healthValue;
+    [SerializeField] private TMP_Text weightValue;
+    [SerializeField] private TMP_Text rangeValue;
+    [SerializeField] private TMP_Text fireValue;
+    [SerializeField] private TMP_Text speedValue;
+    [SerializeField] private TMP_Text maneuvValue;
+
+    [Header("UI Buttons")]
+    [SerializeField] private Button bodyR;
+    [SerializeField] private Button bodyL;
+    [SerializeField] private Button turretR;
+    [SerializeField] private Button turretL;
+    [SerializeField] private Button cannonR;
+    [SerializeField] private Button cannonL;
+    [SerializeField] private Button wheelR;
+    [SerializeField] private Button wheelL;
+
+    private int bodyIndex = 0;
+    private int cannonIndex = 0;
+    private int turretIndex = 0;
+    private int wheelIndex = 0;
+    private void Start()
+    {
+        bodyIndex = 0;
```

Comment:
These variables are already initialized with 0 above, no need to reassign them here.